packages/terra-pills/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@cerner/terra-pills",
+  "main": "lib/index.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "The terra-pills component provides a way to either display a static text or optionally allow trigger actions to launch popovers with the desired content.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git",
+    "directory": "packages/terra-pills"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-pills",
+    "Pills",
+    "Pill",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "files": [
+    "lib",
+    "src"
```

Comment:
If we're using the [files](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#files) field for packing, we should also include the translations. See [example](https://github.com/cerner/terra-application/blob/terra-application%401.52.2/packages/terra-application/package.json#L26-L31).

Reply:
Added 51a9979